Makefile

```diff
@@ -121,8 +127,20 @@ update-schema:
 
 #
 # update autogenerated files
-#
+# (moq is also used but which version is unclear & manual changes made!)
 
 .PHONY: generate
 generate:
-	@go generate ./...
+	@go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.8.2
+	@go install github.com/pseudo-su/oapi-ui-codegen/cmd/oapi-ui-codegen@v0.0.2
+
+	@go generate cmd/dp-find-insights-poc-api/generate.go
+
+#
+# check no manual commits to auto-generated files on a clean tree
+#
+
+.PHONY: check-generate
+check-generate: generate
+	  @[ "$$(git diff)" = "" ] || (echo "commits to autogen file?" && exit 1)
```

Comment:
could the error message for this be made any more specific? (which file, etc)

Reply:
discussed offline, OOS

Reply:
https://github.com/ONSdigital/dp-find-insights-poc-api/issues/124